cmd/decorator/main.go

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
what's the copyright insoncistency about? seems like we should be consistent :)

Reply:
fixed.